tests/test_webui.py

```diff
@@ -44,22 +45,21 @@ def client():
     client.close()
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture
 def random_port():
     # Get random number for ephemeral port, container and image name
     # Typical values from /proc/sys/net/ipv4/ip_local_port_range
     return random.randint(32768, 60999)
 
 
-@pytest.fixture(scope="session")
```

Comment:
why remove session scope?

Reply:
> why remove session scope?

I'm not sure why I added it in the first place, but the `tmp_path` fixture refused to work with it enabled.  The default scope  is ok since we're not reusing the fixtures defined in this test.